### PR TITLE
Added Stimulus to project

### PR DESCRIPTION
**Pull request by GCdM**

### CODE REVIEW

File: app/models/application_record.rb [hunk 1]
```diff
@@ -1,3 +1,12 @@
 class ApplicationRecord < ActiveRecord::Base
   self.abstract_class = true
+
+  def self.encoded_find(encoded_id)
```
Comment (danielkaczmarczyk) [inline app/models/application_record.rb:4]: we've got some mixed space widths, I see 4 and 2

File: app/javascript/controllers/answer_controller.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
+import { Controller } from 'stimulus'
+
+export default class extends Controller {
+
+    connect() {
+        
+    }
+}
```
Comment (danielkaczmarczyk) [inline app/javascript/controllers/answer_controller.js:8]: re: style

**Comment by mandables** [comment]
@danielkaczmarczyk Don't know why I can't reply to your other comment but I've used rubocop and can set it up

**Comment by danielkaczmarczyk** [comment]
@GCdM why close this pr?

**Pull request closed without merging**